mozcloud-workload-core/library/values.yaml.example

```diff
@@ -0,0 +1,69 @@
+# Example values showing a visual representation of the parameters accepted by
+# this library chart.
+
+# This is used to populate the default value for "target" for ExternalSecret
+# resources.
+appCode: tenant-name
+
+# This is used to populate the default value for "gsm.secret" for ExternalSecret
+# resources.
+environment: dev
+
+# If you set this, it will be the default name for all resources if not
+# specified at the resource level.
```

Comment:
Could we add context on why you would do this, and if you don't do this what name various resources will get?